src/clarity-angular/utils/_theme.dark.clarity.scss

```diff
@@ -0,0 +1,694 @@
+/*!
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+// - Global
+// - Alerts
+// - Badge
+// - Button
+// - Card
+// - Datagrid
+// - Dropdown
+// - Form, Input, Checkbox, Radio, Toggle
+// - Header
+// - Label
+// - Login
+// - Modal
+// - Nav
+// - Table
+// - Progress Bars
+// - Signpost
+// - Spinner
+// - Stack View
+// - Table
+// - Tabs
+// - Tooltip
+// - Tree View
+// - Typography
+// - Vertical Nav
+// - Wizard
+
+/*********
+ * Global variables will have an effect on or in multiple Clarity components.
+ * Where possible, there is a list files affected by the variable.
+ */
+
+/*********
+ * $clr-global-app-background
+ * Change the background for the clarity application.
+ * Usage: clarity-root/src/clarity-angular/layout/main-container/_layout.clarity.scss
+ * Affects:
+ * - $clr-login-background-color
+ * - $clr-stack-view-bg-color
+ * - $clr-stack-view-stack-block-label-and-content-bg-color
+ */
+$clr-global-app-background: #1b2a32;
+
+/*********
+ * $clr-global-selection-color
+ * Components that have a selected (or active) state use this to change the background-color when an item or element
+ * is in the active state.
+ * Usages:
+ * - _layout.clarity.scss
+ * - _datagrid.clarity.scss
+ * - _sidenav.clarity.scss
+ *
+ * Component's with variables affected in the dark theme (this file)
+ * - Stack View
+ * - Tree View
+ * - Dropdown
+ */
+$clr-global-selection-color: #324f61;
+
+/*********
+ * $clr-global-hover-bg-color
+ * Components or elements that can be hovered will have this color applied to their background-color.
+ * Component's with variables affected in the dark theme (this file)
+ * - Dropdown
+ * - Tree View
+ */
+$clr-global-hover-bg-color: #29414e;
+
+/*********
+ * The close class is a utility class that can be applied to elements with a clr-icon thats used for closing something.
+ * There are two variables that control the normal and hovered states.
+ * Usages:
+ * - _close.clarity.scss
+ *
+ * Component's with variables affected in the dark theme (this file)
+ * - ??
+ */
+$close-color--normal: #c3ced4;
+$close-color--hover: $clr-white;
+// END close
+
+/*********
+ * Name: Link (and link state colors)
+ * Description the color for links and the active, hovered and visited state
+ * Usages: ./_reboot.clarity.scss
+ */
+$clr-link-active-color: #49afd9;
```

Comment:
Some of these colors are reused in the file, should we hoist them up into local variables?

Reply:
For now I would like to keep them as is b/c we are still tweaking the design a bit. WHen I first started I made variables that tried to semantically group the color usage into a single variable but there has been some churn that has made it difficult to maintain.